inc/head.php

```diff
@@ -207,7 +207,13 @@ function loadKeyboardFromHash() {
     }
     if(typeof addKeyboards == 'function') {
       addKeyboards();
-    }
+    } <?php if ($site_suffix == ".local") {
+      echo 'else {
+      console.warn("-- Fallback:  not using the server\'s cached keyboard set! --");
+      // Server caching may not be active in local dev instances of the server.
+      keyman.addKeyboards();
+    }';
+    } ?>
```

Comment:
Kept having to manually change this for testing, so I figured I'd throw it in for `.local` mode only.

Reply:
OK I guess? Not sure I'm understanding why this is happening?

Reply:
No keyboards show when I use my local version of the page unless I make this change.  I noticed that the `addKeyboards` called by the `if` portion refers to caching query results, which my local setup doesn't seem to facilitate.

Reply:
I feel like there is another problem happening here for which this is a symptom. It's probably not all that important though so go ahead with this :grin: